NetScaler/Public/Disable-NSLBServer.ps1

```diff
@@ -87,7 +92,17 @@ function Disable-NSLBServer {
                         if ($Graceful -gt 0) {
                             $params.Add('delay', $Graceful)
                         } else {
-                            $params.Add('delay', 0)
+                            if ($PSBoundParameters.ContainsKey('Delay')) {
```

Comment:
You don't need this check as `[Int]` parameters are always 0 unless otherwise specified. You should just be able to safely use `$params.Add('delay', $Delay)`

Reply:
I simplified this section now that the Graceful and Delay are mutually exclusive.